templates/rhpam70-sit.yaml

```diff
@@ -897,6 +894,8 @@ objects:
             value: "${MAVEN_REPO_USERNAME}"
           - name: MAVEN_REPO_PASSWORD
             value: "${MAVEN_REPO_PASSWORD}"
+          - name: MAVEN_REPO_PATH
```

Comment:
Are these changes in rhpam70-sit.yaml needed? They seem to enforce using Business Central as maven repo for the template. I think it should be possible to use any maven repo here.

Reply:
I dind't understand your point, can you please clarify?

Reply:
MAVEN_REPO_PATH is only necessary if you are specifying a MAVEN_REPO_SERVICE (or HOST + PORT), because we need a to build a url. For example, if the maven repo service is business central, then specifying a maven repo path allows the code to build a url. However, since here we're only specifying MAVEN_REPO_URL - which defines the **entire** URL already so we don't have to build a url - having MAVEN_REPO_PATH is unnecessary. I would suggest removing the MAVEN_REPO_PATHs.